client/state/ui/selectors.js

```diff
@@ -52,13 +52,23 @@ export function getSelectedSiteSlug( state ) {
 }
 
 /**
+ * Returns the current section.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Object}        Current section
+ */
+export function getSection( state ) {
+	return state.ui.section || {};
```

Comment:
Ah, that makes sense. I previously defaulted it to an object because of the way the other selectors (like `getSectionName`) are accessing it. They would throw an error if the section is not an object. 

But now I've defaulted it to `false` and modified the other selectors to consider that possibility.